met-api/src/met_api/resources/survey.py

```diff
@@ -0,0 +1,104 @@
+# Copyright © 2021 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""API endpoints for managing an survey resource."""
+
+from flask import request
+from flask_cors import cross_origin
+from flask_restx import Namespace, Resource
+
+from met_api.auth import auth
+from met_api.schemas.survey import SurveySchema
+from met_api.services.survey_service import SurveyService
+from met_api.utils.action_result import ActionResult
+from met_api.utils.token_info import TokenInfo
+from met_api.utils.util import allowedorigins, cors_preflight
+
+
+API = Namespace('survey', description='Endpoints for Survey Management')
+"""Custom exception messages
+"""
+
+
+@cors_preflight('GET,OPTIONS')
+@API.route('/<survey_id>')
+class GetSurvey(Resource):
```

Comment:
GetSurvey is a verb and ideally class name should be Noun..
so Survey is easier and GET/PUT can go in here...

Reply:
Updated all resources